setup.py

```diff
@@ -1,9 +1,40 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright 2013-2022 The Foundry Visionmongers Ltd
-from setuptools import setup, find_packages
+from distutils.command.build_py import build_py
+import logging
+import os
+from shutil import copyfile
+from setuptools import setup
+
+import openassetio_traitgen
+
+# The python sources don't exist in this repo, but are generated via
+# openassetio-traitgen at point of build.
+# We generate them directly into the build directory structured as a
+# package.
+class GenerateThenBuild(build_py):
+    def run(self):
+        # Generate traits into a python package under python directory.
```

Comment:
[fix] Is this comment out of date?

Reply:
Yep.

Reply:
[e4c5c21](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/15/commits/e4c5c2112fb46195e518aa82076f8f3afe5b3141)